### PR TITLE
Playground tutorial: add tasks + fixed additional docs

### PR DESCRIPTION
**Pull request by ikoevska**
* Updated tutorial with simple input code
* Updated docs for TextField
* Updated docs for GridLayout

### CODE REVIEW

File: content/docs/en/elements/layouts/grid-layout.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+  <Label text="1,0" row="1" col="0" backgroundColor="#289062"/>
   <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
-  <Label text="1,2" row="1" col="2" backgroundColor="#289062"/>
-  <Label text="2,1" row="2" col="1" backgroundColor="#1c6b48"/>
-  <Label text="2,2" row="2" col="2" backgroundColor="#43b883"/>
 </GridLayout>
 ```
-<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout.svg" />
+
+![Simple GridLayout](/screenshots/simple-grid-layot.jpg)
```
Comment (rigor789) [inline content/docs/en/elements/layouts/grid-layout.md]: You can change it back to `https://art.nativescript-vue.org/layouts/grid_layout.svg` I have updated the graphic in the art repo.

File: content/docs/en/elements/layouts/grid-layout.md [hunk 2]
```diff
@@ -51,23 +55,43 @@ The GridLayout is a layout that arranges its child elements in a table structure
...
+  <Label text="0,0" row="0" col="0" backgroundColor="#43b883"/>
+  <Label text="0,1" row="0" col="1" colSpan="2" backgroundColor="#1c6b48"/>
+  <Label text="1,0" row="1" col="0" rowSpan="2" backgroundColor="#289062"/>
+  <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
+  <Label text="1,2" row="1" col="2" backgroundColor="#289062"/>
+  <Label text="2,1" row="2" col="1" backgroundColor="#1c6b48"/>
+  <Label text="2,2" row="2" col="2" backgroundColor="#43b883"/>
+</GridLayout>
+```
+<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout.svg" />
```
Comment (rigor789) [inline content/docs/en/elements/layouts/grid-layout.md]: You can change it to `https://art.nativescript-vue.org/layouts/grid_layout_complex.svg`

File: content/docs/en/elements/layouts/grid-layout.md [hunk 3]
```diff
@@ -51,23 +55,43 @@ The GridLayout is a layout that arranges its child elements in a table structure
...
 
 ## Props
 
-| name | type | description |
+| Name | Type | Description |
 |------|------|-------------|
-`columns` | `String` | A string value representing column widths delimited with commas. Column widths can be either an absolute number, `auto` or `*`. A number indicates an absolute column width, `auto` makes the column as wide as its widest child, and `*` makes the column occupy all available horizontal space.
```
Comment (rigor789) [inline content/docs/en/elements/layouts/grid-layout.md]: I think we should leave this part in:

> `auto` makes the column as wide as its widest child, and `*` makes the column occupy all available horizontal space

File: content/docs/en/elements/layouts/grid-layout.md [hunk 4]
```diff
@@ -1,34 +1,38 @@
...
 title: GridLayout
 apiRef: https://docs.nativescript.org/api-reference/modules/_ui_layouts_grid_layout_
-contributors: [rigor789]
+contributors: [rigor789, ikoevska]
 ---
 
-The GridLayout is a layout that arranges its child elements in a table structure of rows and columns. A cell can contain multiple child elements, they can span over multiple rows and columns, and even overlap each other. The GridLayout has one column and one row by default. To add additional columns and rows, you have to specify column definition items (separated by commas) to the `columns` property and row definition items (separated by commas) to the `rows` property of the GridLayout. The width of a column and the height of a row can be specified as an absolute amount of pixels, as a percentage of the available space or automatically:
+`<GridLayout>` is a layout component that lets you arrange its child elements in a table-like manner. 
 
-- **Absolute**: Fixed size of pixels.
```
[the remark this answers is not in the document]
Comment (rigor789) [reply content/docs/en/elements/layouts/grid-layout.md]: This includes the 2 other ones, but I couldn't extend my comment across multiple lines

[the remark this answers is not in the document]
Comment (ikoevska) [reply content/docs/en/elements/layouts/grid-layout.md]: Reflected several lines below.

File: content/docs/en/elements/layouts/grid-layout.md (hunk 3 above)
Comment (ikoevska) [reply content/docs/en/elements/layouts/grid-layout.md]: Applied. Appears as change several lines below.

File: content/docs/en/elements/layouts/grid-layout.md [hunk 5]
```diff
@@ -49,25 +58,39 @@ The GridLayout is a layout that arranges its child elements in a table structure
   <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
 </GridLayout>
 ```
-<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg" />
+<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_complex.svg" />
```
Comment (rigor789) [inline content/docs/en/elements/layouts/grid-layout.md]: This has been changed by accident, should be `https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg`

File: content/docs/en/elements/layouts/grid-layout.md [hunk 6]
```diff
@@ -1,34 +1,43 @@
...
+
+### Examples
+
+#### Grid layout with fixed sizing
+
+The following example creates a simple 2-by-2 grid with fixed column widths and row heights.
 
 ```html
-<GridLayout columns="40, auto, *" rows="40, auto, *" backgroundColor="#3c495e">
+<GridLayout columns="150, 150" rows="150, 150">
```
Comment (rigor789) [inline content/docs/en/elements/layouts/grid-layout.md]: Change these to `115,115` to stay consistent with other measures (the image displayed is 230 units wide)

**Comment by ikoevska** [comment]
I've just applied the latest changes from comments :). Thanks for the review and the help!

**Comment by rigor789** [comment]
I know this one was a bit more work, thanks for sticking with it! :)

**Pull request merged**